similarity-data-api/src/main/java/org/phenotips/data/similarity/PatientSimilarityView.java

```diff
@@ -70,4 +70,20 @@
      *         match, with {@code 0} for patients with no similarities
      */
     double getScore();
+
+    /**
+     * Get the phenotypic similarity score for this patient match.
+     *
+     * @return the similarity score, between 0 (a poor match) and 1 (a good
```

Comment:
Scores are never negative, so we could consider it deprecated. I think the concept of an "opposite" patient description is rather tenuous.